### ISSUE TITLE
Regression: vue-apollo `options` argument should be typed optional in useMutate Function

### ISSUE THREAD

**Issue by JoeSchr** [dotansimha/graphql-code-generator#5053]
from docs: 

> options: (default: null) Options object. Can also be a function returning an options object.

https://v4.apollo.vuejs.org/api/use-mutation.html 

working on a fix...

**Comment by dotansimha** [comment]
Thank you for reporting @JoeSchr , Does this also solves that? https://github.com/dotansimha/graphql-code-generator/pull/5076

**Comment by JoeSchr** [comment]
@dotansimha No, `useMutate` gets at least 2 parameters `variables` and `options`. #5054 fixes `options` and #5076 fixes `variables`

Sorry, I didn't have time to fix these hardcoded types in the tests yet

**Comment by dotansimha** [comment]
Got it, thanks @JoeSchr . Feel free to ping me when it's ready for a review :)
